db/migrate/20160322070219_devise_create_users.rb

```diff
@@ -1,4 +1,4 @@
-class DeviseCreateUsers < ActiveRecord::Migration
+class DeviseCreateUsers < ActiveRecord::Migration[4.2]
```

Comment:
Not a blocker, but ann changes to migrations should have been dealt with in a separate PR.